pkg/auth/handler/reset_password_test.go

```diff
@@ -0,0 +1,136 @@
+package handler
+
+import (
+	"testing"
+
+	"github.com/skygeario/skygear-server/pkg/auth/dependency/provider/password"
+	"github.com/skygeario/skygear-server/pkg/auth/response"
+	coreAudit "github.com/skygeario/skygear-server/pkg/core/audit"
+	"github.com/skygeario/skygear-server/pkg/core/auth/authinfo"
+	"github.com/skygeario/skygear-server/pkg/core/auth/authtoken"
+	"github.com/skygeario/skygear-server/pkg/server/audit"
+	"github.com/skygeario/skygear-server/pkg/server/skyerr"
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestResetPasswordHandler(t *testing.T) {
```

Comment:
We can separate the `TestResetPasswordHandler` into `TestResetPasswordRequestPayload` and `TestResetPasswordHandler`. It can help the test run faster, and break it as separate logical test unit.